integration/test.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from retrying import retry
+import time
+import unittest
+import uuid
+
+class CookTest(unittest.TestCase):
+
+    @retry(stop_max_delay=120000, wait_fixed=5000)
+    def wait_for_job(self, job_id, status):
+        job = requests.get('%s/rawscheduler?job=%s' % (self.cook_url, job_id))
```

Comment:
We can make a single session for all of the tests by using https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp